test/lib/rules/whitespace/whitespace.js

```diff
@@ -14,7 +14,12 @@ var addPragma = wrappers.addPragma;
 var userConfig = {
   "custom-rules-filename": null,
   "rules": {
-    "whitespace": true
+	"whitespace": true,
+	"conditionals-whitespace": true,
+	"comma-whitespace": true,
+	"operator-whitespace": true,
+	"semicolon-whitespace": true,
+	"function-whitespace": true,
```

Comment:
I would prefer to be explicit in the configuration and mark the one we aren't using as false.  This way if default changes (e.g., if it gets merged into mainline) configuration is still correct.